src/lib/containers/download_list_v2/CreatePackageV2.tsx

```diff
@@ -7,12 +7,14 @@ import {
 import {
   createPackageFromDownloadListV2,
   getFileHandleByIdURL,
+  createManifestFromDownloadListV2
 } from '../../utils/SynapseClient'
 import { useSynapseContext } from '../../utils/SynapseContext'
 import { DownloadListPackageResponse } from '../../utils/synapseTypes/DownloadListV2/DownloadListPackageResponse'
+import { Checkbox } from '../widgets/Checkbox'
 
 export type CreatePackageV2Props = {
-  onPackageCreation: (zipFileUrl: string) => void
+  onPackageCreation: () => void
```

Comment:
Open URL in CreatePackageV2 instead (but still call back to the parent to update the UI)